tests/app/v1_basic_test.py

```diff
@@ -19,7 +19,7 @@ def test_get_test(app):
     assert "test_data_key" in data 
     assert data["test_data_key"] == "test_data_value" 
 
-def test_post_core_data(app):
+def test_post_core_data(app, headers):
```

Comment:
can we also add a small test that a POST without authentication will fail?

Reply:
Good idea. I had a general test for that in auth_test but I added a test specifically for this endpoint